apps/datahub/src/app/app.component.ts

```diff
@@ -2,32 +2,34 @@ import { Component } from '@angular/core'
 import { ColorService } from '@geonetwork-ui/util/shared'
 import { MdViewFacade } from '@geonetwork-ui/feature/search'
 import { map, switchMap } from 'rxjs/operators'
-import { of } from 'rxjs'
+import { combineLatest, of } from 'rxjs'
 
 @Component({
   selector: 'gn-ui-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
-  $isRecordOpened = this.mdViewFacade.isPresent$
+  isPresentOrLoading$ = combineLatest([
+    this.mdViewFacade.isPresent$,
+    this.mdViewFacade.isLoading$,
+  ])
+  isRecordOpened$ = this.isPresentOrLoading$.pipe(
+    map(([present, loading]) => present || loading)
+  )
 
-  $breadcrumb = this.mdViewFacade.isPresent$.pipe(
-    switchMap((present) =>
-      present
-        ? this.mdViewFacade.metadata$.pipe(
-            map((md) => `Jeu de donnée > ${md.title}`)
-          )
-        : of('Recherche')
-    )
+  breadcrumb$ = this.isPresentOrLoading$.pipe(
+    switchMap(([present, loading]) => {
+      if (present)
+        return this.mdViewFacade.metadata$.pipe(
+          map((md) => `Jeu de donnée > ${md.title}`)
+        )
+      if (loading) return of('Chargement...')
+      return of('Recherche')
```

Comment:
translation

Reply:
I haven't registered any translation here for now since this is all considered placeholder, hope that makes sense...